templates/repo/issue/view_title.tmpl

```diff
@@ -25,13 +25,18 @@
 	{{end}}
 
 	{{if .Issue.IsPull}}
+		{{ $baseTargetTag := printf "<a href=\"%s/src/branch/%s\">%s</a>" .RepoLink (EscapePound .BaseTarget) .BaseTarget }}
```

Comment:
This should be probably also partly be moved to translation, with full url being passed as param

Reply:
I considered that, but I wanted consistentcy with the methodology for the head url. Is there a better way to achieve this? I figure if both the anchor tags are in one place, it prevents accidentally changing one and not the other.